### ISSUE TITLE
[Bug]: Accessibility checker onRunComplete after all hook runs when accessibility tests are skipped, resulting in test failures

### ISSUE THREAD

**Issue by EmilyRocks** [IBMa/equal-access#744]
### Project

cypress accessibility checker

### Browser

Chrome

### Operating system

Windows

### Description

When running a subset of tests that do not include accessibility tests, the after all hook runs for each spec file and results in false test failure due to the error below. This occurs when any kind of filtering is used on the Cypress tests to exclude the accessibility tests: grepping at runtime, using it.only, using it.skip for accessibility tests, running a separate spec file from the accessibility specs in the Cypress UI by filtering, etc. 
Is there a way to avoid running this hook when the accessibility tests are not part of the test run? 
([Cypress recommends not using after all hooks](https://docs.cypress.io/guides/references/best-practices#Using-after-or-afterEach-hooks))

Cypress version 9.5.1
Cypress-Accessibility-Checker version 3.1.12

Error

"after all" hook for "name of test":
CypressError: cy.task('accessibilityChecker') failed with the following error: Cannot read property 'onRunComplete' of undefined at Object.onRunComplete (C:\Example\tests\node_modules\cypress-accessibility-checker\src\lib\ACTasks.js:69:30) at onRunComplete (C:\ Example\tests\node_modules\cypress-accessibility-checker\src\tasks.js:40:20) at module.exports (C:\ Example\tests\node_modules\cypress-accessibility-checker\src\tasks.js:127:20)

### Steps to reproduce

You can repeat this by using Cypress only for a non-accessibility test in the same spec file as the accessibility tests, or by running an unrelated spec file
```
    it.only('Non-Accessibility Test In Test Suite That Also Has Cypress-Accessibility-Checker', () => {
        cy.title().should('eq', 'Verify This and Pass without Running After All Hook')
    })
```

**Comment by tombrunet** [comment]
Will be fixed in 3.1.25
